transports/sqs/index.md

```diff
@@ -79,3 +79,28 @@ The SQS transport uses the default [retries and timeouts](http://docs.aws.amazon
 | `ReadWriteTimeout` | 300s          |
 
 NOTE: NServiceBus will perform [immediate](/nservicebus/recoverability/#immediate-retries) and [delayed](/nservicebus/recoverability/#delayed-retries) retries in addition to retries performed internally by the SQS client.
+
+
+
+
+## Troubleshooting
+
+
+### AmazonSQSException: Request is throttled
+
+When sending messages it can happen that your sending is throttled. When that happens the following exception is logged.
+```
+2017-11-14 23:10:24,314|ERROR|18|NServiceBus.Transports.SQS.MessageDispatcher|Exception from Send.
+Amazon.SQS.AmazonSQSException: Request is throttled. ---> Amazon.Runtime.Internal.HttpErrorResponseException: The remote server returned an error: (403) Forbidden. ---> System.Net.WebException: The remote server returned an error: (403) Forbidden.
+```
+
+When this happens while receiving a message then this can safely be ignored as (recoverability features](/nservicebus/recoverability/) should resolve this. If you are sending messages outside of a receiving message context you will have to be be aware that your send can fail. Especially when sending messages in bulk, specifically in parallel, you can get throttled if the sender has enough resources to suddenly flood the AmazonSQS infrastructure.
+
+A storage query that reads 100,000 items/records and for each a message gets produces and then sending these in parallel is vulneralble to get throttled.
+
+- Limit the amount of concurrent send tasks
```

Comment:
I think this is way to specific with the numbers and mixes vocabulary like parallel and concurrent. When I ran the handler code analysis I could show that mostly the semaphore approach is slower than useful batching.

Also not sure if it is feasible to link to a gist which is not maintained as part of this repo. I'll try to rework this a bit as soon as I have more time

Reply:
I have absolutely no capacity right now. Withdrawing myself from this PR. Maybe someone else from @Particular/aws-maintainers can chime in or we'll get back when we finally have time. Thanks @ramonsmits for raising it